.travis.yml

```diff
@@ -12,7 +12,7 @@ env:
   matrix:
     - LUA=5.2 LUANAME=lua5.2 DO_COVERAGE=coveralls
     # Lua 5.3 isn't available in Ubuntu Trusty, so some magic below installs it.
-    - LUA=5.3 LUANAME=lua5.3 LUALIBRARY=/usr/lib/liblua.so DO_COVERAGE=codecov AWESOME_THEME=xresources
```

Comment:
i would leave this and the `if` condition (also later when i will prepare a `gtk` theme it will be good to set it for other travis profile) but i'm fine with with moving sed to `run.sh`